package.json

```diff
@@ -47,6 +47,10 @@
     "create-react-class": "15.5.3",
     "creditcards": "2.1.2",
     "cross-env": "4.0.0",
+    "d3-array": "1.2.0",
+    "d3-scale": "1.0.6",
+    "d3-selection": "1.1.0",
+    "d3-shape": "1.2.0",
```

Comment:
I had it in initially but we decided to not use it - https://github.com/Automattic/wp-calypso/pull/14623#pullrequestreview-41390000